lightly/api/patch.py

```diff
@@ -0,0 +1,58 @@
+import logging
```

Comment:
The name of the file is a bit weird, as it only patches the swagger `Configuration`.

Reply:
I originally put different patches in there. Now only a single one is left. But I would keep the name if we add more patches in the future.